src/test/java/com/urswolfer/gerrit/client/rest/http/projects/ProjectsParserTest.java

```diff
@@ -78,4 +84,17 @@ public void testParseSingleProjectInfo() throws Exception {
             .withState(ProjectState.ACTIVE)
             .get());
     }
+
+    @Test
+    public void testGenerateProjectInput() throws Exception {
+        ProjectInput projectInput = new ProjectInput();
+        projectInput.name = "MyProject";
+        projectInput.description = "This is a demo project.";
+        projectInput.submitType = SubmitType.CHERRY_PICK;
+        projectInput.owners = Collections.singletonList("MyProject-Owners");
+
+        String actualProjectInput = Files.readFile(getFile("project_input.json")).replaceAll("\n","");
+        Truth.assertThat(projectsParser.generateProjectInput(projectInput))
```

Comment:
> I don't understand what you mean here; I mocked out the parser in the client test.

I was just not sure if this test is required at all since it does not test much more than Gson itself. But of course it also doesn't hurt to have it if it works stable. ;)